pkg/insight/insightstore/applicationcountstore.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/pipe-cd/pipecd/pkg/insight"
 )
 
+type ApplicationCountStore interface {
```

Comment:
How about `ApplicationStore` because I think we typically have 2 types of metrics: Application (ApplicationCount) and Deployment (Deployment Frequency, ...)

Reply:
This `ApplicationCountStore` is only for ApplicationCount, and I am planning to create something like a `deploymentStore` separately from it. And together they are the `insightStore`.
The reason for the separation is that it is not necessary to access the deployment store when processing application metrics, and vice versa.